server/endpoint/health/searcher/endpoint.go

```diff
@@ -77,23 +79,39 @@ func (e *Endpoint) Encoder() kithttp.EncodeResponseFunc {
 
 func (e *Endpoint) Endpoint() kitendpoint.Endpoint {
 	return func(ctx context.Context, request interface{}) (interface{}, error) {
-		var serviceRequest searcher.Request
-		{
-			clusterID, ok := request.(string)
-			if !ok {
-				return nil, microerror.Mask(badRequestError)
-			}
-			serviceRequest = searcher.Request{
-				ClusterID: clusterID,
-			}
+		clusterID, ok := request.(string)
+		if !ok {
+			return nil, microerror.Mask(badRequestError)
 		}
 
-		serviceResponse, err := e.service.Health.Searcher.Search(ctx, serviceRequest)
+		clusterRequest := cluster.Request{
+			ClusterID: clusterID,
+		}
+		clusterResponse, err := e.service.Cluster.Searcher.Search(ctx, clusterRequest)
```

Comment:
I am not super happy with the naming for `Cluster` here - this is information gathered from CR's from the controlplane. That should be more clear somehow.

Reply:
I renamed it to `Host`.